apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -111,25 +116,111 @@
       case 'lockscreen-inputappclosed':
         this.updatePassCodeUI();
         break;
+      case 'touchstart':
+        // TODO: ignore while on error timeout
+        touch = evt.changedTouches[0];
+        target = document.elementFromPoint(touch.clientX, touch.clientY);
```

Comment:
Is it possible to use `evt.target` instead of querying element from the coordination?

https://developer.mozilla.org/en-US/docs/Web/Events/touchstart